subsys/net/lib/lwm2m_client_utils/CMakeLists.txt

```diff
@@ -14,8 +14,8 @@ zephyr_library_sources_ifdef(CONFIG_LWM2M_CLIENT_UTILS_DEVICE_OBJ_SUPPORT lwm2m/
 zephyr_library_sources_ifdef(CONFIG_LWM2M_CLIENT_UTILS_SECURITY_OBJ_SUPPORT lwm2m/lwm2m_security.c)
 zephyr_library_sources_ifdef(CONFIG_LWM2M_CLIENT_UTILS_SIGNAL_MEAS_INFO_OBJ_SUPPORT
 			     lwm2m/ecid_signal_meas_info.c)
-zephyr_library_sources_ifdef(CONFIG_LWM2M_CLIENT_UTILS_NEIGHBOUR_CELL_LISTENER
-			     lwm2m/neighbour_cell_info.c)
+zephyr_library_sources_ifdef(CONFIG_LWM2M_CLIENT_UTILS
+			     lwm2m/lwm2m_lte_notification.c)
```

Comment:
Changing this dependency causes some issues as the `CONFIG_LWM2M_CLIENT_UTILS_NEIGHBOUR_CELL_LISTENER` is dependent of `CONFIG_LWM2M_CLIENT_UTILS_SIGNAL_MEAS_INFO_OBJ_SUPPORT`
The code shouldn't try to populate the neighbour cell information to object if the object itself isn't supported.

Reply:
good point I change lwm2m_lte_notification() to use CONFIG_LWM2M_CLIENT_UTILS_NEIGHBOUR_CELL_LISTENER